www/front_src/src/components/breadcrumbWrapper/Breadcrumb.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
```

Comment:
Can you add the prop-types dependency? then you'll need to commit the updated package.json and package-lock.json

Reply:
@CoralieRubin package-lock.json file is still missing. This file should always come with a change in a package.json because it fixes the package versions for the build.

https://docs.npmjs.com/configuring-npm/package-lock-json.html